tools/bazar/actions/BazarListeAction.php

```diff
@@ -68,6 +68,11 @@ public function formatArguments($arg)
                 $color = $this->params->get('baz_marker_color');
             }
         }
+        $agenda = $arg['agenda'] ?? null ;
+        $champ = $arg['champ'] ?? 'bf_titre' ;
+        $ordre = $arg['ordre'] ?? 'asc' ;
+        // for agenda automatically decreasing order if order not defined
+        $ordre = (!empty($agenda) && $champ != 'bf_date_debut_evenement') ? 'desc': $ordre ;
```

Comment:
c'est pas plutot : 
```php
if ($agenda) {
    // default to date ordered chronologically
    $champ = $arg['champ'] ?? 'bf_date_debut_evenement' ;
    $ordre = $arg['ordre'] ?? 'asc' ;
} else {
    // sorted alphabetically otherwise
    $champ = $arg['champ'] ?? 'bf_titre' ;
    $ordre = $arg['ordre'] ?? 'asc' ;
}
```

Reply:
J'ai fait une correction avec un nouveau commit. Normalement ça prend tout en compte.